src/TestSuite/ConsoleIntegrationTestTrait.php

```diff
@@ -144,6 +144,16 @@ public function useCommandRunner(): void
         $this->_useCommandRunner = true;
     }
 
+    /**
+     * Get an array of messages from \Cake\TestSuite\Stub\ConsoleOutput::messages()
+     *
+     * @return array
+     */
+    public function getMessages(): array
```

Comment:
If we are going to add this method then better to name is `getOutMessage()`.

Reply:
Would it be better just to provide a getter for `\Cake\TestSuite\Stub\ConsoleOutput`?

Reply:
Well we could just use the protected property `$this->_out` directly too.

Whether we need a public interface for the ConsoleOutput instance can be discussed in a separate issue/PR.

Reply:
No, getMessages() is pointless. Not sure why I thought `_out` was private. I removed it and just accessed `_out` directly.